lib/mi_bridges/driver.rb

```diff
@@ -32,6 +32,7 @@ class Driver
       JobIncomeInformationPage,
       JobIncomeMoreAboutPage,
       JobIncomeSummaryPage,
+      CashOnHandPage,
```

Comment:
alpha order?

Reply:
If we alpha here, then we should alpha in each of these arrays. If that's cool w/you then I'll do it in a commit here.